libs/wordpress-plugin/plugin/trunk/block/edit-or-preview/edit/service-callbacks.tsx

```diff
@@ -126,17 +126,6 @@ export const constructServiceModuleCallbacks = (
         displayToast,
       ),
 
-    /** @todo: remove this when `@blockprotocol/service` is updated */
-    openaiCompleteText: async ({ data }) =>
-      callService(
-        {
-          providerName: "openai",
-          methodName: "completeText",
-          data,
-        },
-        displayToast,
-      ),
-
```

Comment:
@CiaranMn I suppose we could re-publish the wordpress plugin to reflect this change? But might not be worth the effort if it's not straightforward, as it doesn't add any functionality and won't improve the experience in any way (just changes where blocks would get an error if they try to call `completeText`)

Reply:
No need... once we publish the latest blocks any instance that adds them won't have it as an option